dockertest/containers.py

```diff
@@ -423,7 +423,7 @@ def kill_container_by_long_id(self, long_id):
         cmd = 'kill '
         if _signal is not None:
             if _signal.upper().startswith('SIG'):
-                _signal = _signal[3:]
```

Comment:
E1136:426,26: DockerContainers.kill_container_by_long_id: Value '_signal' is unsubscriptable